atom/api.py

```diff
@@ -1,10 +1,13 @@
 #------------------------------------------------------------------------------
-# Copyright (c) 2013, Nucleic Development Team.
+# Copyright (c) 2013-2017, Nucleic Development Team.
 #
 # Distributed under the terms of the Modified BSD License.
 #
 # The full license is in the file COPYING.txt, distributed with this software.
 #------------------------------------------------------------------------------
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
```

Comment:
Usually I prefer to systematically include those imports to be sure nothing breaks unexpectedly. But it add noise to this PR so I removed them as there are not exactly necessary.